src/plugins/administrative_templates/ui/presentationbuilder.cpp

```diff
@@ -140,9 +141,64 @@ class PresentationBuilderPrivate : public PresentationWidgetVisitor
+
+                        if (!(*m_stateEnabled)) {
+                            return;
+                        }
+                        qWarning() << "Presentation builder::save: " << elementInfo.key.c_str()
+                                   << " " << elementInfo.value.c_str();
+
+                        // If has one of list.
+                        if (booleanElement->hasFalseList || booleanElement->hasTrueList) {
+                            // Select the appropriate list
+                            const auto &setList = checkBox->checkState() == Qt::Checked
+                                    ? booleanElement->trueList
+                                    : booleanElement->falseList;
+
+                            for (const auto &booleanValue : setList) {
+                                // If the key is not present, then select the one to which
+                                // elementInfo is linked
+                                const std::string &key =
+                                        booleanValue.has_key ? booleanValue.key : elementInfo.key;
+
+                                qWarning() << "Presentation builder::save: " << key.c_str() << " "
+                                           << booleanValue.value_name.c_str();
+
+                                switch (booleanValue.type) {
+
+                                case BooleanValue::BOOLEAN_VALUE_TYPE_DELETED:
+                                    m_source->clearValue(key, booleanValue.value_name);
+                                    break;
+
+                                case BooleanValue::BOOLEAN_VALUE_TYPE_DECIMAL:
+                                    m_source->setValue(key, booleanValue.value_name,
+                                                       RegistryEntryType::REG_DWORD,
+                                                       booleanValue.decimal);
+                                    break;
+
+                                case BooleanValue::BOOLEAN_VALUE_TYPE_LONGDECIMAL:
+                                    m_source->setValue(key, booleanValue.value_name,
+                                                       RegistryEntryType::REG_QWORD,
+                                                       booleanValue.long_decimal);
+                                    break;
+
+                                case BooleanValue::BOOLEAN_VALUE_TYPE_STRING:
+                                    m_source->setValue(key, booleanValue.value_name,
+                                            RegistryEntryType::REG_SZ,
+                                                       QString::fromStdString(booleanValue.string));
+                                    break;
+                                }
+                            }
+                            /// Needed set key.value from elementInfo to true false?
                     return;
                 }
                 qWarning() << "Presentation builder::save: " << elementInfo.key.c_str() << " "
```

Comment:
Add support for trueValue and falseValue.

Reply:
160-199 if `trueList`/`falseList` is present we call one of `setValue` (182, 188, 194) or `clearValue` (178 on DELETE value) for everyone `trueValue`/`falseValue` in list(if checked in `trueList`, otherwise in `falseList`).

Reply:
Specification states that besides `trueList`\`falseList` there are `trueValue`\`falseValue` inside `booleanElement`.
```xml
<boolean id               = string
         clientExtension? = GUID
         key?             = registryKey
         valueName?       = registryValueName >

  <!-- Child elements -->
  trueValue?,
  falseValue?,
  trueList?,
  falseList?

</boolean>
```
On line 161 I see if that covers `trueList`/`falseList` but I do not see any code that supports `trueValue`/`falseValue`.